### ISSUE TITLE
--interactive, --Interactive-start, and other launch options.

### ISSUE THREAD

**Issue by Pasajeromoronmoreno** [antimatter15/alpaca.cpp#68]
I was playing with several of these options, but they seem to do nothing, or I don't know how to use them.
--interactive
--Interactive-start
--reverse-prompt PROMPT
--prompt PROMPT
--file FNAME

I was hoping to make it work in chat mode with some of these options, or to give it a fixed prompt that reads as context before giving a response, but nothing I tried seems to have any effect. If anyone knows how to use them, could you please explain it here, so it can be used as a reference? Thank you!!

**Comment by linonetwo** [comment]
Seems these options are broken, it become `interactive` by default, and ignore the `prompt`

**Comment by Alexmakes** [comment]
ive found the new -f option doesn't seam to work and without a verbose option when loading the model i can not confirm whether or not it is working 
command used
`./chat -m ggml-alpaca-13b-q4.bin -t 39 --color -c 10400 -n 4096 --repeat_last_n 10400 -f cake`

**Comment by GrahamboJangles** [comment]
Yeah I don't think any of these launch options are doing anything

**Comment by VicFic2006** [comment]
Yeah, `--reverse-prompt` is not doing anything for me either.
It would've been great to chat with something like Bob from llama.cpp with a customised personality.
